corio.py

```diff
@@ -68,19 +70,22 @@
 }
 
 
-def initialize_loghandler(level=logging.INFO):
+def initialize_loghandler(opt):
     """
     Initialize io driver runner logging with stream and file handlers.
 
-    param level: logging level used in CorIO tool.
+    param opt.logging_level: logging level used in CorIO tool.
```

Comment:
Change `opt.logging_level` to opt and use `initialize_loghandler(opt.logging_level)`